packages/dx-react-scheduler/api/dx-react-scheduler.api.ts

```diff
@@ -167,6 +167,7 @@ namespace Appointments {
   interface AppointmentContentProps {
     children: React.ReactNode;
     data: object;
+    formatDate: (date: Date, options: any) => string;
```

Comment:
Is there a reason why `FormatDateFn` is not introduced?

Reply:
Few arguments